pkg/scanners/cloud/aws/scanner_test.go

```diff
@@ -62,6 +62,7 @@ func Test_AWSInputSelectors(t *testing.T) {
 	testCases := []struct {
 		name            string
 		srcFS           fs.FS
+		dataFS          fs.FS
```

Comment:
I see you've added this field but haven't defined any cases in this test. I think we should move the test you added as a test case here as most of the logic is common between the two.

Reply:
We are adding to these options in scanner
options.ScannerWithDataFilesystem(tc.dataFS),
options.ScannerWithDataDirs("config-data/")

so we will have to include config data this in all tests

Reply:
You can make the options on the test cases itself rather than declared once.

Reply:
Any update?